### PR TITLE
Implement missing parts of the code

### PR DESCRIPTION
**Pull request by ElahehFarokhi**

### CODE REVIEW

File: Examples/Interfaces/INotificationSender.cs
Comment (mohammadKarimi) [inline Examples/Interfaces/INotificationSender.cs:1]: why this interface is empty? do you want to use it as marker ?

File: Examples/Interfaces/ITicketEvent.cs
Comment (mohammadKarimi) [inline Examples/Interfaces/ITicketEvent.cs:1]: as same as previous comment.

**Review by mohammadKarimi — COMMENTED** [review]
well done. but it needs more code to be complete.

File: Examples/Interfaces/INotificationSender.cs
Comment (ElahehFarokhi) [reply Examples/Interfaces/INotificationSender.cs:1]: I just thought the purpose of this exercise is to show the concept of the pattern.
I fixed it.

**Review by mohammadKarimi — COMMENTED** [review]
well done.

**Pull request closed without merging**